vizkit3d_world.orogen

```diff
@@ -1,15 +1,19 @@
 name "vizkit3d_world"
 
+import_types_from "base"
 import_types_from "vizkit3d_worldTypes.hpp"
 
 using_library "vizkit3d_world"
 
-
 task_context "Task" do
     needs_configuration
     
+    property "model_paths", "/std/vector</std/string>"
     property "world_file_path", "/std/string"
+    property "show_gui", "bool"
```

Comment:
And document each property / port. Just add a comment on top of it, e.g.

# Paths to resolve the model:// URIs in the world file
property "model_paths", ...